src/ts/contentScripts/dispatcher/index.ts

```diff
@@ -18,4 +19,12 @@ browser.runtime.onMessage.addListener((command) => dispatchMap.get(command)?.())
 
 document.addEventListener('keyup', ev => {
     if (ev.key === guard) replaceFuzzyDate();
-});
\ No newline at end of file
+});
+
+
+document.addEventListener('keyup', ev => {
```

Comment:
Ah that's what you meant by dispatcher. I think we should still go with manifest things until #16 is ready

Reply:
Also demo probably should not have a permanent shortcut :)

Reply:
problem with manifest is it creates browser-level shortcuts, which conflict with other tabs (though there's probably a way to limit it to a match just like the contentScripts?). 

also given the constraints seen in #6, manifest shortcuts are a pain to set manually, so this way is also quicker for testing, which was all it was

Reply:
I agree that manifest thing has a variety of limitations, I believe long term we want to go with solution in #16. Until then my preference though is to keep things in manifest, as it's user-configurable and a unified solution.

Reply:
deleted shortcut, used solution from #23 for testing purposes